daemons/attrd/attrd_commands.c

```diff
@@ -785,8 +785,8 @@ attrd_peer_update(crm_node_t *peer, xmlNode *xml, const char *host, bool filter)
     }
 
     // NULL because PCMK__ATTRD_CMD_SYNC_RESPONSE has no PCMK__XA_TASK
-    update_both = ((op == NULL)
-                   || safe_str_eq(op, PCMK__ATTRD_CMD_UPDATE_BOTH));
+    update_both = pcmk__str_eq(op, PCMK__ATTRD_CMD_UPDATE_BOTH,
+                               pcmk__str_null_matches | pcmk__str_casei);
```

Comment:
Just as an aside, we've lazily used safe_str_eq() in lots of places where matches should really be case-sensitive. I dread the day when we need to go over all the uses and figure out what we should change now, what we should change at a compatibility break, and what we should leave alone. Thankfully today is definitely not that day ...